pkg/kubelet/status/status_manager.go

```diff
@@ -480,7 +492,7 @@ func (m *manager) RemoveOrphanedStatuses(podUIDs map[types.UID]bool) {
 	defer m.podStatusesLock.Unlock()
 	for key := range m.podStatuses {
 		if _, ok := podUIDs[key]; !ok {
-			klog.V(5).Infof("Removing %q from status map.", key)
+			klog.V(5).InfoS("Removing key from status map.", "key", key)
```

Comment:
In this case key is the pod UID.
```suggestion
			klog.V(5).InfoS("Removing pod from status map.", "podUID", key)
```